src/include/daos_srv/container.h

```diff
@@ -129,6 +130,12 @@ struct ds_cont_child {
 	 */
 	uint64_t		sc_ec_update_timestamp;
 
+	/*
+	 * The gap between the max allowed aggregation epoch and current HLC. The modification
+	 * with older epoch out of range may cause conflict with aggregation as to be rejected.
+	 */
+	uint64_t		 sc_agg_eph_gap;
```

Comment:
Isn't it a global value? I don't see why it's per-container.

Reply:
Put it in `ds_cont_child` just for using it easily when aggregation. I will consider a dss variable to replace that.